### ISSUE TITLE
Let's talk about payment audio

### ISSUE THREAD

**Issue by ty-everett** [gatewaycash/gateway#77]
The current default audio when a PayButton transaction completes is https://github.com/gatewaycash/gateway/blob/master/modules/website/public/audio/ding.mp3 but there are other choices, such as those proposed by the BCA.

Having a standard audio clip across all BCH apps is important for augmenting the payment experience for customers. Please feel free to link to other sounds which are licensed friendly to the project and I'll look at getting them implemented.

If we want it to be standard we also might consider removing or deprecating the `paymentCompleteAudio` attribute of PayButton so that the same audio gets played regardless of where the button is used. I'd love to hear thoughts on this as well, it would be trading customization for standardization which is an interesting trade-off.

**Comment by m4ktub** [comment]
>  [...] it would be trading customization for standardization which is an interesting trade-off.

I don't think that what's in question is a trade-off. The ability to customize (including disabling sound) and differentiate is important. Nevertheless, users expect an out-of-the-box solution that just works, that is, that works within the expectations of both the seller and the buyer. Having the "right" sound that buyers recognize as a successful payment is, in my opinion, gateway.cash's responsibility or, in other words, should be part of the provided service.

Currently I would propose:
1. Changing the semantics of `paymentCompleteAudio` to allow keywords or URLs,
2. Have named sound clips like "gateway", "bca", etc for convenience, and 
3. Interpret `off` or `none` as no sound.

This would make it simpler to disable, choose, or customize the audio sample. Consequently, it should allow more experimentation. It still leaves out the question of the default value. In that case my biased opinion is to default to "bca" or the [receive sound](https://github.com/The-Bitcoin-Cash-Fund/Branding/blob/master/Bitcoin_Cash/Audio/BCH_Payment_Receive_Audio.wav) provided in 2017 by the Bitcoin Cash Association

**Comment by ty-everett** [comment]
This makes so much sense, I didn't ever think to combine `enablePaymentAudio` and `paymentCompleteAudio` for some reason. Working on this now, thank you for this. I agree about the default value too.
